External/Plugins/ResultsPanel/ListViewEx.cs

```diff
@@ -1,94 +1,71 @@
 ﻿using PluginCore;
-using PluginCore.Helpers;
-using PluginCore.Managers;
 using System;
 using System.Collections.Generic;
 using System.Drawing;
-using System.Drawing.Imaging;
-using System.Linq;
-using System.Text;
 using System.Windows.Forms;
 using System.Windows.Forms.VisualStyles;
 
 namespace ResultsPanel
 {
-    class ListViewEx : System.Windows.Forms.ListViewEx
+    internal class ListViewEx : System.Windows.Forms.ListViewEx
```

Comment:
I see the `internal` on this line is only about code style.
But the members are kind of different when it comes to reflection, because the members are non-public. And I think it is better to keep every access modifier to as restricted as possible, as they aren't designed to be accessible (class is marked internal).